spec/models/pacbio/run_spec.rb

```diff
@@ -6,32 +6,33 @@
   let!(:version10) { create(:pacbio_smrt_link_version, name: 'v10', default: true) }
 
   context 'uuidable' do
-    let(:uuidable_model) { :pacbio_run }
+    let(:uuidable_model) { :pacbio_revio_run }
 
     it_behaves_like 'uuidable'
   end
 
   context 'validation' do
-    it 'must have a sequencing kit box barcode' do
-      expect(build(:pacbio_run, sequencing_kit_box_barcode: nil)).not_to be_valid
+    it 'does not need a sequencing kit box barcode' do
+      expect(build(:pacbio_sequel_run, sequencing_kit_box_barcode: nil)).to be_valid
     end
 
     it 'must have a DNA control complex kit box barcode' do
-      expect(build(:pacbio_run, dna_control_complex_box_barcode: nil)).not_to be_valid
+      expect(build(:pacbio_sequel_run, dna_control_complex_box_barcode: nil)).not_to be_valid
     end
 
     it 'must have a system name' do
-      expect(build(:pacbio_run, system_name: nil)).not_to be_valid
+      expect(build(:pacbio_sequel_run, system_name: nil)).not_to be_valid
     end
 
     context 'when system name is Revio' do
       it 'does not need a DNA control complex barcode' do
-        expect(build(:pacbio_run, system_name: 'Revio', dna_control_complex_box_barcode: nil)).to be_valid
+        expect(build(:pacbio_revio_run, dna_control_complex_box_barcode: nil)).to be_valid
```

Comment:
Pacbio revio run does also not need `sequencing_kit_box_barcode`. For tests shared before both sequel and revio runs, maybe they could be refactored

Reply:
@stevieing There were a few comments that were "hidden", not sure if you saw them too